captain/utils/pytest/discover_tests.py

```diff
@@ -10,14 +10,37 @@
 import pytest
 from pytest_jsonreport.plugin import JSONReport
 from captain.utils.import_utils import unload_module
+import re
 
 
-def discover_pytest_file(path: str, one_file: bool, return_val: list):
+def check_missing_imports(report: JSONReport):
+    missing_lib = set()
+    if not report.report:
+        return missing_lib
+    if "collectors" not in report.report:
+        return missing_lib
+    for element in report.report["collectors"]:
```

Comment:
best not to use as a dict, but rather access like a class object (for developer experience)